package.json

```diff
@@ -45,7 +45,7 @@
   "scripts": {
     "lintfix": "eslint --fix ./ && prettier --write '**/*.{js,json,md,yaml,yml}'",
     "lint": "eslint ./",
-    "test": "jest --forceExit",
+    "test": "jest --forceExit --maxWorkers=3",
```

Comment:
> Why do they have to be async?

Why not in the top and async? Even if we use Babel imports should be done at the top


> Because of [facebook/jest#10662 (comment)](https://github.com/facebook/jest/issues/10662#issuecomment-923850177), as explained on [#1026 (comment)](https://github.com/iterative/cml/pull/1026#issue-1250268830)

Its not CML enforcing node 14.x?

Reply:
And what's the difference?

Reply:
Its not that only affected by node 12.x?

Reply:
And they don't run in Node.js 12